lahja/base.py

```diff
@@ -395,6 +390,20 @@ def get_connected_endpoints_and_subscriptions(
         """
         ...
 
+    @property
+    @abstractmethod
+    def is_local_broadcast_enabled(self) -> bool:
+        ...
+
+    @abstractmethod
+    def enable_local_broadcast(self) -> None:
+        """
+        When enabled, this endpoint will treat itself as one of it's connected
+        remote endpoints when dealing with event broadcasting and
+        subscriptions.
+        """
```

Comment:
I think this docstring sums up what I don't like about this change: **When enabled** (some core APIs behave very different depending on the fact of the feature is enabled/disabled) (...) *treat itself as one of it's connected remote endpoints* -> To me feels like some APIs are just telling lies in subtle ways. *Maybe* some of this could be changed with API renames though (e.g. `is_any_remote_subscribed_to` -> `is_any_endpoint_subscribed_to`)